Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM stvstnfrd/workbench:latest
+MAINTAINER stv <stv@stanford.edu>
+ADD package.json /root/xblock/
+RUN cd /root/xblock && npm install
```

Comment:
Lines 3-4 are optional, but helpful.
By copying over the `package.json` and installing the node packages "early",
we're able to cache the install as a docker layer.
`package.json` is unlikely to change often, whereas the following step on line 5
will frequently bust the cache (if any file in the project directory is modified).
This way we already have the packages installed, so subsequent attempts can NOOP.